formats/json/commonTest/src/kotlinx/serialization/features/DurationTest.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017-2022 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.features
+
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonTestBase
+import kotlin.test.Test
+import kotlin.time.Duration
+import kotlin.time.DurationUnit
+import kotlin.time.toDuration
+
+
+class DurationTest : JsonTestBase() {
```

Comment:
I think it makes sense to have two tests: one commented that have `DurationHolder` and checks that plugin correctly bounds serializer (existing now), and the other, uncommented, that explicitly uses `Duration.serializer()` and verifies that actual serializer works correctly.

Reply:
Added separated test on `DurationSerializer` to `BasicTypesSerializationTest` but left these test.
Also added extra check to commented test in `SerializersLookupTest`.